otherlibs/stdune/bytes_unit.ml

```diff
@@ -0,0 +1,37 @@
+let conversion_table =
+  [ ([ "bytes"; "B" ], 1L)
+  ; ([ "KB"; "kB"; "KiB"; "kilobytes" ], 1000L)
```

Comment:
`KiB` is not 1000, it's 1024. Using `MiB` and `GiB` below isn't accurate either.